lib/Echidna/Exec.hs

```diff
@@ -66,23 +66,26 @@ vmExcept e = throwM $ case VMFailure e of {Illegal -> IllegalExec e; _ -> Unknow
 -- | Given an error handler, an execution function, and a transaction, execute that transaction
 -- using the given execution strategy, handling errors with the given handler.
 execTxWith :: (MonadState x m, Has VM x) => (Error -> m ()) -> m VMResult -> Tx -> m (VMResult, Int)
-execTxWith h m t = do (og :: VM) <- use hasLens
-                      setupTx t
-                      gasIn <- use $ hasLens . state . gas
-                      res <- m
-                      gasOut <- use $ hasLens . state . gas
-                      cd  <- use $ hasLens . state . calldata
-                      case (res, t ^. call) of
-                        (f@Reversion, _)            -> do hasLens .= og
-                                                          hasLens . state . calldata .= cd
-                                                          hasLens . result ?= f
-                        (VMFailure x, _)            -> h x
-                        (VMSuccess bc, SolCreate _) -> (hasLens %=) . execState $ do
-                          env . contracts . at (t ^. dst) . _Just . contractcode .= InitCode ""
-                          replaceCodeOfSelf (RuntimeCode bc)
-                          loadContract (t ^. dst)
-                        _                        -> pure ()
-                      return (res, fromIntegral (gasIn - gasOut))
+execTxWith h m t = do
+  (og :: VM) <- use hasLens
```

Comment:
I always found confusing the `og` name, where it was easier to use `vm` or something like that.

Reply:
good point, I will change this to `vm`